<!DOCTYPE html>
<html><head><title>_logger</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">1 write to _logger</div><div class="rA">OrchardCore.Users (1)</div><div class="rG" id="OrchardCore.Users"><div class="rF"><div class="rN">Controllers\AccountController.cs (1)</div>
<a href="../Controllers/AccountController.cs.html#70"><b>70</b><i>_logger</i> = logger;</a>
</div>
</div>
<div class="rH">32 references to _logger</div><div class="rA">OrchardCore.Users (32)</div><div class="rG" id="OrchardCore.Users"><div class="rF"><div class="rN">Controllers\AccountController.cs (32)</div>
<a href="../Controllers/AccountController.cs.html#147"><b>147</b><i>_logger</i>.LogError(ex, &quot;An error occurred while validating DefaultExternalLogin token&quot;);</a>
<a href="../Controllers/AccountController.cs.html#202"><b>202</b>await _accountEvents.InvokeAsync((e, model, modelState) =&gt; e.LoggingInAsync(model.UserName, (key, message) =&gt; modelState.AddModelError(key, message)), model, ModelState, <i>_logger</i>);</a>
<a href="../Controllers/AccountController.cs.html#219"><b>219</b><i>_logger</i>.LogInformation(1, &quot;User logged in.&quot;);</a>
<a href="../Controllers/AccountController.cs.html#220"><b>220</b>await _accountEvents.InvokeAsync((e, model) =&gt; e.LoggedInAsync(model.UserName), model, <i>_logger</i>);</a>
<a href="../Controllers/AccountController.cs.html#230"><b>230</b>await _accountEvents.InvokeAsync((e, model) =&gt; e.LoggingInFailedAsync(model.UserName), model, <i>_logger</i>);</a>
<a href="../Controllers/AccountController.cs.html#242"><b>242</b><i>_logger</i>.LogInformation(4, &quot;User logged out.&quot;);</a>
<a href="../Controllers/AccountController.cs.html#356"><b>356</b><i>_logger</i>.LogError(ex, &quot;Error Syncing Roles From External Provider {0}&quot;, info.LoginProvider);</a>
<a href="../Controllers/AccountController.cs.html#369"><b>369</b><i>_logger</i>.LogError(ex, &quot;{externalLoginHandler} - IExternalLoginHandler.UpdateRoles threw an exception&quot;, item.GetType());</a>
<a href="../Controllers/AccountController.cs.html#386"><b>386</b><i>_logger</i>.LogError(&quot;Error updating the external authentication tokens.&quot;);</a>
<a href="../Controllers/AccountController.cs.html#399"><b>399</b><i>_logger</i>.LogError(&quot;Error from external provider: {Error}&quot;, remoteError);</a>
<a href="../Controllers/AccountController.cs.html#407"><b>407</b><i>_logger</i>.LogError(&quot;Could not get external login info.&quot;);</a>
<a href="../Controllers/AccountController.cs.html#419"><b>419</b>await _accountEvents.InvokeAsync((e, user, modelState) =&gt; e.LoggingInAsync(user.UserName, (key, message) =&gt; modelState.AddModelError(key, message)), user, ModelState, <i>_logger</i>);</a>
<a href="../Controllers/AccountController.cs.html#456"><b>456</b><i>_logger</i>.LogWarning(message);</a>
<a href="../Controllers/AccountController.cs.html#485"><b>485</b>}, S[&quot;Confirm your account&quot;], <i>_logger</i>);</a>
<a href="../Controllers/AccountController.cs.html#493"><b>493</b><i>_logger</i>.LogInformation(3, &quot;User account linked to {LoginProvider} provider.&quot;, info.LoginProvider);</a>
<a href="../Controllers/AccountController.cs.html#542"><b>542</b><i>_logger</i>.LogWarning(&quot;Error loading external login info.&quot;);</a>
<a href="../Controllers/AccountController.cs.html#548"><b>548</b><i>_logger</i>.LogWarning(&quot;Site does not allow user registration.&quot;, model.UserName, model.Email);</a>
<a href="../Controllers/AccountController.cs.html#580"><b>580</b>user = await this.RegisterUser(new RegisterViewModel() { UserName = model.UserName, Email = model.Email, Password = model.Password, ConfirmPassword = model.ConfirmPassword }, S[&quot;Confirm your account&quot;], <i>_logger</i>);</a>
<a href="../Controllers/AccountController.cs.html#590"><b>590</b><i>_logger</i>.LogInformation(3, &quot;User account linked to {provider} provider.&quot;, info.LoginProvider);</a>
<a href="../Controllers/AccountController.cs.html#620"><b>620</b><i>_logger</i>.LogWarning(&quot;Error loading external login info.&quot;);</a>
<a href="../Controllers/AccountController.cs.html#626"><b>626</b><i>_logger</i>.LogWarning(&quot;Suspicious login detected from external provider. {provider} with key [{providerKey}] for {identity}&quot;,</a>
<a href="../Controllers/AccountController.cs.html#631"><b>631</b>await _accountEvents.InvokeAsync((e, model, modelState) =&gt; e.LoggingInAsync(user.UserName, (key, message) =&gt; modelState.AddModelError(key, message)), model, ModelState, <i>_logger</i>);</a>
<a href="../Controllers/AccountController.cs.html#644"><b>644</b><i>_logger</i>.LogInformation(3, &quot;User account linked to {provider} provider.&quot;, info.LoginProvider);</a>
<a href="../Controllers/AccountController.cs.html#695"><b>695</b><i>_logger</i>.LogError(&quot;Unable to load user with ID &#39;{UserId}&#39;.&quot;, _userManager.GetUserId(User));</a>
<a href="../Controllers/AccountController.cs.html#702"><b>702</b><i>_logger</i>.LogError(&quot;Unexpected error occurred loading external login info for user &#39;{UserName}&#39;.&quot;, user.UserName);</a>
<a href="../Controllers/AccountController.cs.html#709"><b>709</b><i>_logger</i>.LogError(&quot;Unexpected error occurred adding external login info for user &#39;{UserName}&#39;.&quot;, user.UserName);</a>
<a href="../Controllers/AccountController.cs.html#728"><b>728</b><i>_logger</i>.LogError(&quot;Unable to load user with ID &#39;{UserId}&#39;.&quot;, _userManager.GetUserId(User));</a>
<a href="../Controllers/AccountController.cs.html#735"><b>735</b><i>_logger</i>.LogError(&quot;Unexpected error occurred removing external login info for user &#39;{UserName}&#39;.&quot;, user.UserName);</a>
<a href="../Controllers/AccountController.cs.html#744"><b>744</b><i>_logger</i>.LogError(&quot;Could not remove &#39;{TokenName}&#39; token while unlinking &#39;{LoginProvider}&#39; provider from user &#39;{UserName}&#39;.&quot;, item.Name, model.LoginProvider, user.UserName);</a>
<a href="../Controllers/AccountController.cs.html#779"><b>779</b><i>_logger</i>.LogError(ex, &quot;Error evaluating GenerateUsernameScript({context})&quot;, context);</a>
<a href="../Controllers/AccountController.cs.html#801"><b>801</b><i>_logger</i>.LogError(ex, &quot;{externalLoginHandler} - IExternalLoginHandler.GenerateUserName threw an exception&quot;, item.GetType());</a>
<a href="../Controllers/AccountController.cs.html#806"><b>806</b><i>_logger</i>.LogWarning(&quot;More than one IExternalLoginHandler generated username. Used first one registered, {externalLoginHandler}&quot;, userNames.FirstOrDefault().Key);</a>
</div>
</div>
</body></html>